gui/wxpython/datacatalog/tree.py

```diff
@@ -682,6 +686,35 @@ def OnCreateMapset(self, event):
                    message=_("Unable to create new mapset: %s") % e,
                    showTraceback=False)
 
+    def OnCreateLocation(self, event):
+        """
+        Location wizard started
+        """
+        grassdatabase, location, georeffile, default_region, user_mapset = \
+        create_location_interactively(self,
+                                      self.selected_grassdb[0].data['name'])
```

Comment:
The correct way how to write this would be:

```python
        grassdatabase, location, georeffile, default_region, user_mapset = (
            create_location_interactively(self, self.selected_grassdb[0].data['name'])
        )
```

However, most likely, it will not be needed since the following code really needs to go to the `create_location_interactively()` function.

Reply:
I formatted it to grassdatabase, location, user_mapset = (
            create_location_interactively(self,
                                          self.selected_grassdb[0].data['name'])
        )